utils/appliance/implementations/ui.py

```diff
@@ -232,7 +232,7 @@ def do_nav(self, _tries=0, *args, **kwargs):
         br = self.appliance.browser
 
         try:
-            self.step()
+            self.step(*args, **kwargs)
```

Comment:
:1st_place_medal: , two beer this man. Thanks. I forgot to push my PR with kwargs when worked on virtual machines.